lib/rspec/core/bisect/example_minimizer.rb

```diff
@@ -8,7 +6,7 @@ module Bisect
       # repeatedly running different subsets of the suite.
       class ExampleMinimizer
         attr_reader :runner, :reporter, :all_example_ids, :failed_example_ids
-        attr_accessor :remaining_ids
+        attr_accessor :eliminated_ids
```

Comment:
Hmm. I'd thought it was because it made calculating the working set for each run easier, but on reflection it doesn't at all. Will revert. :)